package.json

```
@@ -22,6 +22,9 @@
 		"ci:build": "npm run build",
 		"ci:lint": "npm run lint",
 		"ci:test": "npm run test-unit:coverage-ci",
+		"dev": "concurrently \"webpack --config webpack.dev.js\" \"npm-watch build:css\"",
+		"dev-w": "webpack --watch --config webpack.dev.js",
```

this would still need to be 

```suggestion
		"dev-w": "concurrently \"webpack --watch --config webpack.dev.js\" \"npm-watch build:css\"",
```

in order to build CSS as well no?

---

I saw that it's different than the old `watch`, but I trusted and copy-pasted that from you :D

---

HA!!

ya don't blindly follow **MY** suggestions!!!